pkg/controller/installation/core_controller.go

```diff
@@ -1402,6 +1412,26 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 		goldmaneRunning = goldmaneCR != nil
 	}
 
+	// Get the Goldmane Service in order to find its cluster IP.
+	goldmaneIP := ""
+	if goldmaneRunning {
```

Comment:
This seems like a useful general utility for this situation, might be worth it to pull this out into a function accessible to all controllers.